src/components/PricingTable/index.js

```diff
@@ -281,6 +282,15 @@ class PricingTable extends Component {
 		this.stripeKey = 'pk_live_ihb1fzO4h1ykymhpZsA3GaQR';
 	}
 
+	componentDidMount() {
+		const plan = getParam('plan', window.location.search) || null;
+		setTimeout(() => {
```

Comment:
I think you don't even need to manage state for it, just a getter function to compute the plan should be enough.

Reply:
Yeah makes sense we won't need state, we can directly compare the params through the getter.

We need `setTimeout` to ensure that the Stripe component has been mounted, otherwise the Modal is not appearing. So for a safe side I have placed half second timeout.